modules/apps/portal-security/portal-security-access-control-impl/src/main/java/com/liferay/portal/security/access/control/internal/advice/AccessControlAdvisorImpl.java

```diff
@@ -62,11 +67,16 @@ public void accept(
 		}
 	}
 
+	@Activate
+	protected void activate() {
+		_accessControlPolicies = ServiceTrackerListFactory.open(
+			SystemBundleUtil.getBundleContext(), AccessControlPolicy.class);
+	}
+
 	private static final Log _log = LogFactoryUtil.getLog(
 		AccessControlAdvisorImpl.class.getName());
 
-	private static final ServiceTrackerList<AccessControlPolicy>
-		_accessControlPolicies = ServiceTrackerListFactory.open(
-			SystemBundleUtil.getBundleContext(), AccessControlPolicy.class);
+	private static ServiceTrackerList<AccessControlPolicy>
```

Comment:
@julschong Since it's a component now, let's turn the field an instance field, and open/close it with activation/deactivation.